File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/PackagesCreatePackagesPlantingsRequestItem.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class PackagesCreatePackagesPlantingsRequestItem(
    @JsonProperty("LocationName")
    val locationName: String? = null,
    @JsonProperty("PackageAdjustmentAmount")
    val packageAdjustmentAmount: Int? = null,
    @JsonProperty("PackageAdjustmentUnitOfMeasureName")
    val packageAdjustmentUnitOfMeasureName: String? = null,
    @JsonProperty("PackageLabel")
    val packageLabel: String? = null,
    @JsonProperty("PatientLicenseNumber")
    val patientLicenseNumber: String? = null,
    @JsonProperty("PlantBatchName")
    val plantBatchName: String? = null,
    @JsonProperty("PlantBatchType")
    val plantBatchType: String? = null,
    @JsonProperty("PlantCount")
    val plantCount: Int? = null,
    @JsonProperty("PlantedDate")
    val plantedDate: String? = null,
    @JsonProperty("StrainName")
    val strainName: String? = null,
    @JsonProperty("SublocationName")
    val sublocationName: String? = null,
    @JsonProperty("UnpackagedDate")
    val unpackagedDate: String? = null
)

File: sdks/thunkmetrc-go/wrapper/models/packages_create_packages_plantings_request_item.go
package models

type PackagesCreatePackagesPlantingsRequestItem struct {
    LocationName string `json:"LocationName,omitempty"`
    PackageAdjustmentAmount int `json:"PackageAdjustmentAmount,omitempty"`
    PackageAdjustmentUnitOfMeasureName string `json:"PackageAdjustmentUnitOfMeasureName,omitempty"`
    PackageLabel string `json:"PackageLabel,omitempty"`
    PatientLicenseNumber string `json:"PatientLicenseNumber,omitempty"`
    PlantBatchName string `json:"PlantBatchName,omitempty"`
    PlantBatchType string `json:"PlantBatchType,omitempty"`
    PlantCount int `json:"PlantCount,omitempty"`
    PlantedDate string `json:"PlantedDate,omitempty"`
    StrainName string `json:"StrainName,omitempty"`
    SublocationName string `json:"SublocationName,omitempty"`
    UnpackagedDate string `json:"UnpackagedDate,omitempty"`
}
